tools/toolchains/arm.py

```diff
@@ -66,6 +66,9 @@ def __init__(self, target, notify=None, macros=None,
         self.asm = [main_cc] + self.flags['common'] + self.flags['asm']
         self.cc = [main_cc] + self.flags['common'] + self.flags['c']
         self.cppc = [main_cc] + self.flags['common'] + self.flags['c'] + self.flags['cxx']
+        if target.name in ["REALTEK_RTL8195AM"]:
```

Comment:
I'm surprised the size of the enum changes linking compatibility, are pointers to these enums being passed around?

If a fixed sized is required, I would suggest just using `uint32_t` where necessary. This documents the requirement on storage size and avoids portability concerns. Passing enums to integers can be implcicitly casted in C, I think it requires a manual cast in C++.

@theotherjimmy's method would work, but is a bit of a hack for insuring binary compatibility. C/C++ don't garuntee the size/alignment of enums, so unfortunately enums simply shouldn't be relied on for portability.